aidants_connect_web/models.py

```diff
@@ -329,11 +332,33 @@ class Mandat(models.Model):
     )
     is_remote = models.BooleanField("Signé à distance ?", default=False)
 
+    template_path = models.TextField(
+        "Template utilisé",
+        null=True,
+        blank=False,
+        default=mandate_template_path,
+        editable=False,
+    )
+
     objects = MandatQuerySet.as_manager()
 
     def __str__(self):
         return f"#{self.id}"
 
+    def save(
+        self, force_insert=False, force_update=False, using=None, update_fields=None
+    ):
+        # Prevent changing the template path
+        if self.pk:
+            try:
```

Comment:
ici tu peux faire plus simple juste faire un if not self.pk:
   alors tu fais un truc 
sinon tu fais rien.

Reply:
Et si quelqu'un passe un paramètre à `template_path` dans le code à l'init ? :thinking: